ext/README.md

```diff
@@ -0,0 +1,78 @@
+---
+tags: oci,distribution
+breaks: false
+---
+
+# Extensions API for Distribution
+
+The basis of the Extension API is described in a document which should
+be emulated for all extensions.
+
+## Table
+
+_notice_: All new `./ext/ext-$name.md` docs MUST be added to this table.
```

Comment:
Was our goal to have a centralized registry of extensions? Or, is this for standardized extensions?

Reply:
I was not expecting this doc to be the clearing or mechanism for registration of extensions but community could submit extensions to this repo. For now I was only considering the discovery extension to be in this location. 
./ext/ext-oci.md which would call out the definition for `/v2/_ext/oci/extensions/list` and `/v2/<repo>/_ext/oci/extensions/list` or similar discovery APIs.